src/selfplay/loop.cc

```diff
@@ -45,7 +46,7 @@ void SelfPlayLoop::RunLoop() {
   options_.Add<BoolOption>(kInteractive, "interactive") = false;
   SelfPlayTournament::PopulateOptions(&options_);
 
-  if (!options_.ProcessAllFlags()) return;
+  if (!ConfigFile::Init(&options_) || !options_.ProcessAllFlags()) return;
```

Comment:
It's hard to say how it should work in selfplay vs uci mode.
It surely should not be the same config (if a person did some tuning for uci mode, and then started training, it should not use the same params), but maybe it makes sense to allow some config (e.g. for gpu config).

For now I think it's safer not to read any configs at all in selfplay mode.

Or... Maybe make default config name uci.config, selfplay.config, etc. RegisterMode and ConsumeCommand can be refactored to store mode, but probably that's better to do later. So, no config for selfplay mode for now.